### PR TITLE
[Suyabay][#106110260] Feature readme

### PR DESCRIPTION
**Pull request by ibonly**

### CODE REVIEW

**Comment by unicodeveloper** [comment]
Please squash into one commit

**Comment by unicodeveloper** [comment]
Please also include the command to clone with the repo url

File: readme.md [hunk 1]
```diff
@@ -3,11 +3,68 @@
...
+        DB_HOST    =localhost
+        DB_DATABASE=
+        DB_USERNAME=
+        DB_PASSWORD=
+
+        ### Test database configuration
+        DB_TEST_HOST   =localhost
+        DB_TEST_DATABASE=
+        DB_TEST_USERNAME=
+        DB_TEST_PASSWORD=
```
Comment (unicodeveloper) [inline readme.md]: Give them values of `xxxxxxx` , then specify that they have to replace the value of `xxxxxx` with real values

File: readme.md [hunk 2]
```diff
@@ -3,11 +3,68 @@
...
+        SENDER_NAME=
+
+## Requirements
+
+        php                   >=  5.5.9
+        laravel/framework      =  5.1.17
+        busayo/laravel-yearly  =  1.0.*
+        guzzlehttp/guzzle      =  ~4.0
+
+## Requirement for Development
```
Comment (unicodeveloper) [inline readme.md]: This should be `Requirements for Development`

File: readme.md [hunk 3]
```diff
@@ -3,11 +3,68 @@
 [![Build Status](https://travis-ci.org/andela/suyabay.svg)](https://travis-ci.org/andela/suyabay)
 [![License](https://poser.pugx.org/andela/suyabay/license.svg)](LICENSE.md)
 
-Suyabay Podcast is an aggregator of podcasts for Suya lovers across the net. The purpose of this site is to help Suya lovers find new podcasts and contents. I encourage you to subscribe to the individual podcasts feeds and support the content creators.
+Suyabay Podcast is an open source project using the full features of `laravel`. The main purpose of this project is to help Suya lovers find new podcasts and contents.
+
+## Installation
+1. Clone the repository into your project folder
```
Comment (unicodeveloper) [inline readme.md:9]: Also write the command to clone with the repo url. 

Note: Always assume that non-programmers see this repo and want to install it to make use of it.

File: readme.md [hunk 4]
```diff
@@ -3,11 +3,68 @@
...
+2. Run composer install from `cmd ` to install all project dependencies
```
Comment (unicodeveloper) [inline readme.md]: `composer install` should be the one highlighted not `cmd` . Now, you also need to let the user know how to install composer if they don't have it yet.

Comment (ibonly) [inline readme.md]: Okay, I will update it.

**Pull request merged**